Apps/WebClient/src/ClientApp/src/components/modal/verifySMS.vue

```diff
@@ -186,7 +187,7 @@ export default class VerifySMSComponent extends Vue {
             resendTime = this.smsResendDateTime;
         }
         resendTime = resendTime.add(this.config.timeouts.resendSMS * 60 * 1000);
-        return resendTime.diff(new DateWrapper());
+        return resendTime.diff(new DateWrapper()).milliseconds;
```

Comment:
is it a bug? if yes, should we get a separate bug to fix it?

Reply:
Leo:  Its ok to fix any bug while working on a ticket.  Just need to document it in the PR or possibly what you're indicating create a bug and reference it in the PR as well.

Reply:
not a bug